doc/cephfs/quota.rst

```diff
@@ -30,7 +27,9 @@ value::
   setfattr -n ceph.quota.max_bytes -v 5Gi /some/dir           # 5 GiB
 
 .. note:: Values will be strictly cast to IEC units even when SI units
-   are input, e.g. 1K to 1024 bytes.
+   are input, e.g. 1K to 1024 bytes. And the values must be aligned to
```

Comment:
s/And/Additionally,

(the comma is intentional)

Reply:
Done.